### PR TITLE
Macro/Micro bombs are now restricted to the Nuclear game mode.

### PR DESCRIPTION
**Pull request by Ikarrus**
Also fixes some broken spans.

### CODE REVIEW

**Comment by phil235** [comment]
:+1:

**Comment by Steelpoint** [comment]
Good change :+1:

**Comment by Xhuis** [comment]
thank christ

**Comment by Miauw62** [comment]
fastmerged on request of ikky because maxcap bombs every round

File: html/changelogs/Ikarrus-fuckbombs.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
```
Comment (Miauw62) [inline html/changelogs/Ikarrus-fuckbombs.yml]: >not removing the template comments

**Comment by Steelpoint** [comment]
That was quick.

**Pull request merged**